packages/components/avatar/src/avatar-group.tsx

```diff
@@ -31,8 +31,8 @@ export const AvatarGroup = forwardRef<AvatarGroupProps, "div">((props, ref) => {
     max,
     borderColor,
     gap = "-sm",
```

Comment:
this should be fixed, but not sure about that...

Reply:
Hi, in this case, it is like this.

```diff
- gap = "-sm"
+ gap = "-0.5rem",
```